core/deployment/src/main/java/io/quarkus/deployment/SuppressForbidden.java

```diff
@@ -1,5 +1,7 @@
 package io.quarkus.deployment;
 
+import org.springframework.core.annotation.AliasFor;
```

Comment:
is there on alternative? might be smell. on the other hands its widely used therefore seems stable.